### PR TITLE
this should be HOST_PATH not CONTAINER_PATH

### PR DESCRIPTION
**Pull request by ebartz**

### CODE REVIEW

File: _posts/2017-01-25-docker-volumes.markdown [hunk 1]
```diff
@@ -328,7 +328,7 @@ docker container run -v HOST_PATH:CONTAINER_PATH [OPTIONS] IMAGE [CMD]
 Note: HOST_PATH and CONTAINER_PATH can be a folder or file. HOST_PATH must exist before running this command.
```
Comment (marcosnils) [inline _posts/2017-01-25-docker-volumes.markdown]: This argument is also incorrect. The `HOST_PATH` doesn't necessarily need to exist before launching the container. Would you mind fixing this as well?

File: _posts/2017-01-25-docker-volumes.markdown [hunk 2]
```diff
@@ -328,7 +328,7 @@ docker container run -v HOST_PATH:CONTAINER_PATH [OPTIONS] IMAGE [CMD]
...
 
 Several cases to consider:
-- the CONTAINER_PATH does not exist within the container
```
Comment (marcosnils) [inline _posts/2017-01-25-docker-volumes.markdown:331]: I believe this case is correct as it is.. In this particular case, the CONTAINER_PATH doesn't exist beforehand in the container

File: _posts/2017-01-25-docker-volumes.markdown (hunk 1 above)
Comment (ebartz) [reply _posts/2017-01-25-docker-volumes.markdown]: done

File: _posts/2017-01-25-docker-volumes.markdown (hunk 2 above)
Comment (ebartz) [reply _posts/2017-01-25-docker-volumes.markdown:331]: I had a couple of students which found it confusing. Maybe lets just drop that whole thing and just keep the sidenote that I just wrote?

**Comment by jlk1980** [comment]
Ok docker runs command,

**Comment by jlk1980** [comment]
Im still trying to share images on comments

**Review by Mellbabe — APPROVED** [review]

**Review by Mellbabe — APPROVED** [review]

**Review by Mellbabe — APPROVED** [review]

**Pull request merged**